src/clients/remote.rb

```diff
@@ -159,8 +158,9 @@ def SetRAHandler(options)
       else
         Remote.Disable
       end
+      success = Remote.Write
 
-      true
+      success
```

Comment:
NP: You can avoid storing the result to `success` variable, you can directly return the `Remote.Write` value without a temporary variable.